### PR TITLE
Update readme to have instructions for running e2e

### PR DESCRIPTION
**Pull request by eyhhuang**
Closes https://github.com/googleinterns/scsi2nvme/issues/179

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,36 @@
...
+
+## End-to-end translation ##
+
+### Setup ###
+1) Call `$ make` in the root directory. Ensure this results in a scsi2nvme.ko
+2) Call `$ insmod scsi2nvme.ko` to insert the kernel module.
+
+    Upon inserting, the SCSI mock device will be automatically attached. Once the device is attached, the SCSI subsystem will issue a set of commands. 
+
+    `$ lsblk ` should show the device at /dev/sdb
```
Comment (harloff-g) [inline README.md:21]: The exact directory at which the device is shown varies based on how many SCSI disks are attached to the machine

File: README.md [hunk 2]
```diff
@@ -4,6 +4,39 @@
...
+    `$ lsscsi` should show the device with the name "NVMe &nbsp; &nbsp; &nbsp; &nbsp;"
+
+### Issuing commands ###
+Issue commands to the SCSI device as such:
+
+Read: 
+>`$ sudo dd if=/dev/sdb bs=$BLOCK_SIZE count=$NUM_BLOCKS skip=STARTING_BLOCK_ADDRESS | hexdump -C`
+
+Write:
+>`$ sudo dd of=/dev/sdb bs=$BLOCK_SIZE count=$NUM_BLOCKS skip=STARTING_BLOCK_ADDRESS | hexdump -C`
```
Comment (basimsahaf) [inline README.md:33]: Should we be mentioning what exactly to write to the disk here?

Comment (eyhhuang) [reply README.md:33]: that's a good point!

File: README.md [hunk 3]
```diff
@@ -4,6 +4,128 @@
...
+### Setup ###
+1) Ensure the following:
+    - an NVMe device is attached to the machine at /dev/nvme0n1
+    - kernel version is >= 4.19
+1) Call `$ make` in the root directory. Ensure this results in a scsi2nvme.ko
+1) Call `$ insmod scsi2nvme.ko` to insert the kernel module.
+
+    Upon inserting, the SCSI mock device will be automatically attached. Once the device is attached, the SCSI subsystem will issue a set of commands. 
+
+    `$ lsscsi` should show the device with the name "NVMe &nbsp; &nbsp; &nbsp; &nbsp;"
```
Comment (Ralith) [inline README.md:105]: What's with all the `&nbsp;`?

Comment (eyhhuang) [reply README.md]: The name is supposed to be NVMe followed by 4 spaces and markdown will just render 4 consecutive spaces as one space. Although I realized that the way I had it gave it 8 spaces. I also added a comment in parenthesis to describe what it is

Comment (Ralith) [reply README.md]: Ah, nice!

**Pull request merged**